.github/scripts/removeIssuesFromDesignProjects.js

```diff
@@ -52,7 +52,7 @@ try {
     const deleteQuery = `mutation { deleteProjectV2Item(input: {projectId: "${projectItem.project.id}", itemId: "${projectItem.id}"}) { clientMutationId } }`;
```

Comment:
Side note: `removeIssuesFromDesignProjects` ➡️ `removeIssueFromDesignProjects` to match the workflow name?